recipe/meta.yaml

```diff
@@ -1,52 +1,100 @@
 {% set name = "smac" %}
-{% set version = "2.1.0" %}
-
+{% set version = "1.2" %}
+{% set git_version = "1.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: af9d3bffd4d8d233ecc872a8c0f7c245043d41340a7790091165d363b2f7fe63
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: f98e2c6baa91dd275ca4f0fedb47a0d38fe5875564a0805b6b1c6b8866561c24
+    patches:
+      - patches/0001-fix-numpy-deprecated-float-to-float64.patch
+      - patches/0002-fix-numpy-deprecated-int-to-int64.patch
+      - patches/0003-n_restarts_optimizer-must-be-positive.patch
+  # use GH release for missing data files
+  - url: https://github.com/automl/SMAC3/archive/refs/tags/v{{ git_version }}.tar.gz
+    sha256: 72d8874ded51abeb0a775dc7056548b0c4a5d82ea74edbb066ddde633437a5ca
+    patches:
+      # same patch here because a test file is patched as well
+      - patches/0002-fix-numpy-deprecated-int-to-int64.patch
+    folder: gh_src
 
 build:
   number: 0
-  noarch: python
+  # skip win because pynisher 0.6.4 unavailable
+  skip: True  # [py<38 or win]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - smac = smac.smac_cli:cmd_line_call
-  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
+    - python
     - pip
-    - python >=3.8
+    - setuptools
+    - wheel
   run:
-    - configspace >=0.6.1
-    - dask-core
-    - dask-jobqueue
-    - regex
-    - numpy >=1.23.3
-    - scipy >=1.9.2
+    - python
+    - numpy >=1.7.1
+    - scipy >=1.7.0
     - psutil
-    - pynisher >=1.0.0
-    - ConfigSpace >=0.6.1
+    - pynisher >=0.4.1
+    - configspace >=0.4.14,<0.5
     - joblib
-    - scikit-learn >=1.1.2
-    - pyrfr >=0.9.0
-    - dask
+    - scikit-learn >=0.22.0
+    - pyrfr >=0.8.0
+    - dask-core
+    - distributed
     - emcee >=3.0.0
-    - python >=3.8
-    - pyyaml
-    - __linux
+
+{% set tests_to_ignore = "" %}
+# ignore test_cli because they require examples folder and run scripts
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_cli/test_restore_state.py" %}
+# ignore test_psmac_facade.py and test_hydra_facade.py because they
+# use a non existing data file (spear_hydra_test_scenario.txt).
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_facade/test_hydra_facade.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_facade/test_psmac_facade.py" %}
+# ignore tests that crash instead of timeout
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_tae/test_exec_func.py" %}  # [osx]
```

Comment:
Will these tests work with `pytest-timeout`?

Reply:
@skupr-anaconda they did not help.